### DISCUSSION TITLE
Azure servers down?

### DISCUSSION THREAD

**Discussion by TheRealOne78** [community/community#44212] [Actions]
Hi. I can't run this workflow because apt can't connect to the azure packages server.

https://status.azure.com/en-us/status reports all services to be up yet this workflow can't connect to it

Workflow: https://github.com/TheRealOne78/XAWP-gui/actions/runs/3910638007/jobs/6683340815#step:3:1

**Comment by RobPasMue** [comment]
Getting the same issue in multiple repositories. @github-staff could you please look into this?

**Comment by RobPasMue** [comment]
Example of failing workflow: https://github.com/pyansys/pyansys/actions/runs/3910735582/jobs/6683304470
